user-service/.env.sample

```diff
@@ -1,12 +0,0 @@
-# To use a cloud database, set ENV=PROD and fill in the DB_CLOUD_URI
```

Comment:
Sorry but can you restore this file back? We are using this as a template to write our own .env file when developing. It's convenient to have this file so we can copy from it